model/reliability/query_test.go

```diff
@@ -67,6 +71,41 @@ func insertDailyTaskStats(project string, requester string, taskName string, var
 	return err
 }
 
+func handleNoFormat(format string, i int) string {
+	n := strings.Count(format, "%")
+	if n > 0 {
+		return fmt.Sprintf(format, i)
+	}
+	return format
+}
+
+func insertManyDailyTaskStats(many int, projectFmt string, requesterFmt string, taskNameFmt string, variantFmt string, distroFmt string, date time.Time, numSuccess, numFailed, numTimeout, numTestFailed, numSystemFailed, numSetupFailed int, avgDuration float64) error {
+
+	items := []interface{}{}
```

Comment:
this is a dumb microptimization, but I'd construct the slice as `make([]interface, 0, many)` which means that the underlying array is allocated with enough space for all of the items, and avoids reallocation